src/NuGet.Core/NuGet.Packaging/PackageExtraction/StreamExtensions.cs

```diff
@@ -2,12 +2,19 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.IO;
+using System.IO.MemoryMappedFiles;
 
 namespace NuGet.Packaging
 {
     public static class StreamExtensions
     {
-        public static string CopyToFile(this Stream inputStream, string fileFullPath)
+
+        /**
+        Only files smaller than this value will be mmap'ed
+        */
+        private const long MAX_MMAP_SIZE = 10 * 1024 * 1024;
+        public static string CopyToFile(this Stream inputStream, string fileFullPath) => CopyToFile(inputStream, fileFullPath, null);
+        public static string CopyToFile(this Stream inputStream, string fileFullPath, long? size)
```

Comment:
Although we don't seem to have anything in our coding guidelines (at least, doing a text search, I didn't find it), normally between class members (methods at least) there's one line of whitespace.